libafl/Cargo.toml

```diff
@@ -247,42 +249,42 @@ tokio = { version = "1.38", optional = true, features = [
   "rt-multi-thread",
   "time",
 ] } # used for TCP Event Manager and multi-machine
-enumflags2 = { version = "0.7", optional = true }
+enumflags2 = { version = "0.7.10", optional = true }
 
-wait-timeout = { version = "0.2", optional = true } # used by CommandExecutor to wait for child process
+wait-timeout = { version = "0.2.0", optional = true } # used by CommandExecutor to wait for child process
 
-concat-idents = { version = "1.1.3", optional = true }
+concat-idents = { version = "1.1.5", optional = true }
 
-libcasr = { version = "2.7", optional = true }
+libcasr = { version = "2.12.1", optional = true }
 
-bitvec = { version = "1.0", optional = true, features = [
+bitvec = { version = "1.0.1", optional = true, features = [
   "serde",
 ] } # used for string range storage
 
-arrayvec = { version = "0.7.4", optional = true, default-features = false } # used for fixed-len collects
+arrayvec = { version = "0.7.6", optional = true, default-features = false } # used for fixed-len collects
 
-const_format = "0.2.32" # used for providing helpful compiler output
-const_panic = "0.2.8"   # similarly, for formatting const panic output
+const_format = "0.2.33" # used for providing helpful compiler output
+const_panic = "0.2.9"   # similarly, for formatting const panic output
 
-pyo3 = { version = "0.22", optional = true }          # For nautilus
-regex-syntax = { version = "0.8.3", optional = true } # For nautilus
+pyo3 = { version = "0.22.3", optional = true }        # For nautilus
+regex-syntax = { version = "0.8.4", optional = true } # For nautilus
 
 # optional-dev deps (change when target.'cfg(accessible(::std))'.test-dependencies will be stable)
-serial_test = { version = "3", optional = true, default-features = false, features = [
+serial_test = { version = "3.1.1", optional = true, default-features = false, features = [
   "logging",
 ] }
 
 # Document all features of this crate (for `cargo doc`)
-document-features = { version = "0.2", optional = true }
+document-features = { version = "0.2.10", optional = true }
 # Optional
-clap = { version = "4.5", optional = true }
+clap = { version = "4.5.18", optional = true }
 
 [target.'cfg(unix)'.dependencies]
-libc = "0.2"                                 # For (*nix) libc
-z3 = { version = "0.12.0", optional = true } # for concolic mutation
+libc = "0.2.159"                             # For (*nix) libc
```

Comment:
If we specify that stuff so closely, we should start using workspace dependencies instead so we don't have multiple crates with different dependency versions